src/Model/Widget/Span.php

```diff
@@ -41,7 +41,7 @@ public function toStyledGraphemes(Style $baseStyle): array
         return array_map(function (string $grapheme) use ($baseStyle) {
             return new StyledGrapheme(
                 symbol: $grapheme,
-                style: $baseStyle->patch($this->style),
+                style: $this->style->patch($baseStyle),
```

Comment:
note the original is faithful to Ratatui: https://github.com/ratatui-org/ratatui/blob/9f371000968044e09545d66068c4ed4ea4b35d8a/src/text/span.rs#L141

opened a PR on your PR https://github.com/KennedyTedesco/php-tui/pull/1

Reply:
Nice! Very well thought out. It makes sense to me. I need to have this mindset to look at Ratatui behavior, since we're based on them.